d3a_api_client/utils.py

```diff
@@ -107,11 +107,11 @@ def get_area_uuid_from_area_name_and_collaboration_id(collab_id, area_name, doma
     return area_uuid
 
 
-def get_area_uuid_and_name_mapping_from_simulation_id(collab_id, domain_name):
+def get_area_uuid_and_name_mapping_from_simulation_id(collab_id):
```

Comment:
Why not keeping both options by making the domain name an optional param? For example:
```python
def get_area_uuid_and_name_mapping_from_simulation_id(collab_id, domain_name=None):
    domain_name = domain_name or domain_name_from_env()
    # ...
```

Reply:
I'm suggesting this because I think that it would be good to be able to pass an alternative value to these methods without having to change env variables.